websites/D/Docchi/presence.ts

```diff
@@ -139,6 +140,7 @@ presence.on("UpdateData", async () => {
 							},
 						];
 					}
+					presenceData.state = "Docchi.pl";
```

Comment:
Not really sure if this is advertising, if it is it isn't allowed.

Reply:
It’s not meant to look like an advertisement. I set it up due to changing the presence name to 'watching series,' which means there's no longer 'Watching Docchi' while watching, and nothing besides this 'Docchi.pl' state indicates the site you're watching on. However, if this is not allowed, I’ll remove it :)

Reply:
It's klnda meh to me...

Reply:
fine, then im gonna remove it